src/openms/include/OpenMS/CHEMISTRY/ModificationsDB.h

```diff
@@ -65,22 +65,29 @@ namespace OpenMS
       the way described at the unimod.org website and download the file then
       from unimod.org. The same can be done to add support for the modifications
       to search engines, e.g. Mascot.
+
+      In some scenarios, it might be useful to define different modification
+      databases. This can be done by providing a path when initializing
+      ModificationsDB.
   */
   class OPENMS_DLLAPI ModificationsDB
   {
 public:
 
     /// Returns a pointer to the modifications DB (singleton)
-    inline static ModificationsDB* getInstance()
+    inline static ModificationsDB* getInstance(OpenMS::String unimod_file = "CHEMISTRY/unimod.xml", OpenMS::String psimod_file = "CHEMISTRY/PSI-MOD.obo", OpenMS::String xlmod_file = "CHEMISTRY/XLMOD.obo")
```

Comment:
This seems to be problematic for pyOpenMS. @hroest, do you know what I need to change in the corresponding pxd to make this work?